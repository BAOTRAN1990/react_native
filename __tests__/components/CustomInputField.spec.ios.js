// Link.react-test.js
import React from 'react';
import InputFieldComponent from 'src/components/CustomInputField.ios.js';
import renderer from 'react-test-renderer';

import constVar from 'src/const/constant';

it('Test name input field', (done) => {
  const component = renderer.create(
    <InputFieldComponent 
        placeholder='User name'
        imageIcon={require('assets/images/user_name.png')}
        secureTextEntry={false} placeholderTextColor={constVar.colors.WHITE}
        onChangeText={(userName) => this.setState({userName})}/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
  done();
});