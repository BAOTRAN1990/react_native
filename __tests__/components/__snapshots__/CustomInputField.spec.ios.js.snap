// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test name input field 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "borderBottomColor": "#808080",
      "borderBottomWidth": 1,
      "flexDirection": "row",
      "height": 50,
      "marginTop": 10,
    }
  }
>
  <Image
    resizeMode="contain"
    source={1}
    style={
      Object {
        "flex": 1,
        "height": 20,
      }
    }
  />
  <TextInput
    autoCapitalize="none"
    autoCorrect={false}
    imageIcon={1}
    onChangeText={[Function]}
    placeholder="User name"
    placeholderTextColor="#FFFFFF"
    secureTextEntry={false}
    style={
      Object {
        "flex": 5,
        "height": 50,
      }
    }
  />
</View>
`;
